app/src/main/java/com/example/coursematchdaddy/clean_architecture_layers/entities/interfaces/survey_class_imports_implementations/UserDataInterface.java

```diff
@@ -1,4 +1,6 @@
 package com.example.coursematchdaddy.clean_architecture_layers.entities.interfaces.survey_class_imports_implementations;
 
+import java.util.ArrayList;
+
 public interface UserDataInterface {
```

Comment:
In the design doc this interface has a method _**`public HashMap<String, Object> getUserSurveyData()`**_
Add that method and implement it in any classes that implement this interface

Reply:
That wasn't supposed to be in this pull req, its for another issue, I changed it by accident.